src/main/java/com/minecolonies/coremod/items/ItemAncientTome.java

```diff
@@ -0,0 +1,20 @@
+package com.minecolonies.coremod.items;
+
+import com.minecolonies.coremod.creativetab.ModCreativeTabs;
+
+/**
+ * Class handling the AncientTome item.
+ */
+public class ItemAncientTome extends AbstractItemMinecolonies
+{
+
+    /**
+     * Sets the name, creative tab, and registers the item.
+     */
+    public ItemAncientTome()
+    {
+        super("ancientTome");
+        super.setCreativeTab(ModCreativeTabs.MINECOLONIES);
+        setMaxStackSize(64);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 64 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)